pandas/tests/indexing/test_datetime.py

```diff
@@ -168,3 +168,21 @@ def test_getitem_str_slice_millisecond_resolution(self, frame_or_series):
             ],
         )
         tm.assert_equal(result, expected)
+
+    def test_getitem_pyarrow_index(self, frame_or_series):
+        # GH 53644
+        pytest.importorskip("pyarrow")
```

Comment:
isnt there a decorator for this?

Reply:
I think there is, but I think that decorator can be eventually replaced with this builtin pytest functionality